ropa/gui/controller/list_widget_controller.py

```diff
@@ -13,12 +13,13 @@ def __init__(self, widget):
 
     def show_in_gadgets_list(self, gadgets):
         self.widget.clear()
-        font = qg.QFont()
-        font.setFamily(_fromUtf8('Courier new'))
-
         for gadget in gadgets:
-            cell = gadget['address'] + '\n'
-            cell += '\n'.join(gadget['instructions']) + '\n'
-            item = qg.QListWidgetItem(qc.QString(cell), self.widget)
-            item.setFont(font)
-            self.widget.insertItem(self.widget.count(), item)
+            cell = '<pre>'
+            cell += '<b>%s</b>\n' % gadget['address']
+            for instruction in gadget['instructions']:
+                cell += '%s\n' % instruction
+            cell += '</pre>'
```

Comment:
I think better if we can create something like a template file, and use perhaps format strings.